modules/ROOT/pages/kafka/kafka-connector-reference.adoc

```diff
@@ -319,60 +319,73 @@ Operation that facilitates Kafka message consumption from a given topic.
+| Key a| String |  |  | x
+| Partition a| Number |  |  | x
+| Offset a| Number |  |  | x
 |===
 
-[[RedeliveryPolicy]]
-=== Redelivery Policy
+[[Offset]]
+=== Offset
 
-[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Max Redelivery Count a| Number | The maximum number of times a message can be redelivered and processed unsuccessfully before triggering process-failed-message |  |
-| Use Secure Hash a| Boolean | Whether to use a secure hash algorithm to identify a redelivered message |  |
-| Message Digest Algorithm a| String | The secure hashing algorithm to use. If not set, the default is SHA-256. |  |
-| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property may only be set if useSecureHash is false. |  |
-| Object Store a| <<ObjectStore>> | The object store where the redelivery counter for each message is going to be stored. |  |
+| Partition Number a| String |  |  | 
+| Partition Offset a| String |  |  | 
 |===
 
 [[repeatable-in-memory-stream]]
 === Repeatable In Memory Stream
 
-[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Initial Buffer Size a| Number | This is the amount of memory that will be allocated in order to consume the stream and provide random access to it. If the stream contains more data than can be fit into this buffer, then it will be expanded by according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  |
-| Buffer Size Increment a| Number | This is by how much will be buffer size by expanded if it exceeds its initial size. Setting a value of zero or lower will mean that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error will be raised when the buffer gets full. |  |
-| Max Buffer Size a| Number | This is the maximum amount of memory that will be used. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error will be raised. A value lower or equal to zero means no limit. |  |
+| Initial Buffer Size a| Number | The amount of memory that will be allocated to consume the stream and provide random access to it. If the stream contains more data than can be fit into this buffer, then the buffer expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  | 
+| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, and to raise a STREAM_MAXIMUM_SIZE_EXCEEDED error if the buffer gets full. |  | 
+| Max Buffer Size a| Number | The maximum amount of memory to use. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower than or equal to zero means no limit. |  | 
```

Comment:
```suggestion
| Max Buffer Size a| Number | The maximum amount of memory to use before a `STREAM_MAXIMUM_SIZE_EXCEEDED` error is raised. A value lower than or equal to zero means there is no maximum limit. |  | 
```

Reply:
I think the correction is incorrect. According the current text, if set to zero or less, means the buffer should not expand. Your correction of no maximum limit doesn't seem correct.